### PR TITLE
MF-566: Fix Trend-line and charts turning black

### PR DESCRIPTION
**Pull request by donaldkibet**
### Description
[ticket MF-566](https://issues.openmrs.org/projects/MF/issues/MF-566?filter=allissues)
At the moment [patient-chart - feature/monorepo](https://github.com/openmrs/openmrs-esm-patient-chart/tree/feature/monorepo) doesn't provide chart styles this would provide the chart styles
### Screenshoot
Screenshot 2021-04-26 at 16 02 59

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,8 @@
     "@babel/preset-env": "^7.11.0",
     "@babel/preset-react": "^7.10.4",
     "@babel/preset-typescript": "^7.10.4",
+    "@carbon/charts": "^0.41.57",
```
Comment (FlorianRappl) [inline package.json:28]: Wrong place. These are now rather dependencies of `esm-styleguide`.

File: package.json [hunk 2]
```diff
@@ -41,6 +43,7 @@
     "css-loader": "^3.2.0",
     "cssnano": "^4.1.10",
     "css-minimizer-webpack-plugin": "^1.2.0",
+    "d3": "5.x",
```
Comment (FlorianRappl) [inline package.json:46]: Same applies here.

Also I don't see any reference to it - so do we really need this?

File: package.json (hunk 1 above)
Comment (donaldkibet) [reply package.json]: Moved this to `esm-styleguide`

File: package.json (hunk 2 above)
Comment (donaldkibet) [reply package.json]: `carbon/charts` uses `d3` as a dependent

Comment (FlorianRappl) [reply package.json]: Yes, but we only use it for stylesheets right now. So are the sheets depending on D3?

File: packages/esm-styleguide/src/_all.scss [hunk 3]
```diff
@@ -1,5 +1,6 @@
 @import "./vars";
 @import "~carbon-components/scss/globals/scss/styles";
+@import "~@carbon/charts/styles/styles.scss";
```
Comment (FlorianRappl) [inline packages/esm-styleguide/src/_all.scss:3]: Are these sheets depending on D3?

File: packages/esm-styleguide/package.json [hunk 4]
```diff
@@ -39,7 +39,10 @@
     "access": "public"
   },
   "dependencies": {
+    "@carbon/charts": "^0.41.57",
+    "@carbon/charts-react": "^0.41.57",
     "carbon-components-react": "7.31.0",
+    "d3": "5.x",
```
Comment (FlorianRappl) [inline packages/esm-styleguide/package.json:45]: Again, not clear why we have / need this.

File: packages/esm-styleguide/package.json [hunk 5]
```diff
@@ -39,7 +39,10 @@
     "access": "public"
   },
   "dependencies": {
+    "@carbon/charts": "^0.41.57",
+    "@carbon/charts-react": "^0.41.57",
```
Comment (FlorianRappl) [inline packages/esm-styleguide/package.json:43]: Why charts-react?

I only see you using `@carbon/charts`.

**Review by FlorianRappl — APPROVED** [review]
LGTM!

**Pull request merged**